Business apps tutorial (#628)
SimpleDataLabsInc/prophecy-docs: docs/analysts/business-apps/run-apps.md

---
title: Run apps
id: run-apps
slug: /analysts/run-apps
description: Learn how to use applications that have been shared with you
tags: []
---

A business app is a visual application built on top of a Prophecy pipeline. When you run a business app, you execute the underlying pipeline using your own parameters.

## Access and sharing

Access to business apps is controlled by Prophecy’s team-based permission model.

> **Teams own projects.** If your team owns a project, you have full edit access. This means that you can build, edit, and delete business apps in the App Builder.

Reviewer: ```suggestion
### Team owned projects

If your team owns a project, you have full edit access. This means that you can build, edit, and delete business apps in the App Builder.
```

Suggestion: Format-wise, I don't love the bolding at the beginning of a paragraph. I think it makes more sense to make it a sub-section.

Applies to the paragraph right after this too.

- Name your config (e.g., `Q1 Sales Report`, `NY Region Data Pull`).
- Customize input values based on your use case.
- Save and reuse the config later.
- Schedule the config to run at defined intervals.

:::note
Configs are stored per team. Other team members can access and run your configs if they belong to the same team.
:::

### Create config

Follow the steps below to create a new config for an app.

1. Navigate to the **App Directory**.
1. Open the business app you want to use.
1. If no configs exist yet, you’ll be prompted to create one. Otherwise:
   - Hover over the app name in the sidebar.
   - Click the **+** icon to add a new config.
1. Fill in the required inputs (form fields that map to pipeline parameters).
1. Click **Run** to execute the app.

## Schedules

You can set each app config to periodically run using a **schedule**.

### Schedule a config

To create and enable an app schedule:

1. Open the business app.
1. Select the config you want to schedule.
1. From the app header, click the **...** options meu.
1. Click **Schedule**.
1. Define the frequency.
1. Turn on the **Enable** toggle.
1. Click **Schedule**.

> Once you have created the schedule, it needs to be **published** to the appropriate fabric (the execution environment that the pipeline will run on). This is the exact same process as [scheduling pipelines](/analysts/scheduling) themselves.

Reviewer: ```suggestion
After you create the schedule, publish it to the appropriate fabric (the execution environment that the pipeline will run on). This is the same process as [scheduling pipelines](/analysts/scheduling) themselves.
```
